foxglove/middleware.py

```diff
@@ -390,8 +390,18 @@ async def get_cloudflare_ips() -> List[IPRangeCounter]:
     """
 
     async def get_ips(v: Literal[4, 6]) -> List[IPRangeCounter]:
-        r = await glove.http.get(f'https://www.cloudflare.com/ips-v{v}')
-        UnexpectedResponse.check(r)
+        try:
+            r = await glove.http.get(f'https://www.cloudflare.com/ips-v{v}')
```

Comment:
```suggestion
            r = await glove.http.get(f'https://www.cloudflare.com/ips-v{v}', follow_redirects=True)
```

According to that tweet, the response was a redirect when it wasn't working. Is that what the logs showed?

If so, this should be more robust to future changes, and you can remove the second try,except block.

Reply:
Yeah, that's right. It was a 301 error. Let me just recode the correct fix on a branch, so I can double-check the fix, before deploying.

Reply:
You can run it locally but commenting out the code that prevents this middleware being installed.

But from a quick curl, it seem like the reverted the change and both with and without `/` now return 200.

Reply:
Yep, `curl -v --location --request GET 'https://www.cloudflare.com/ips-v4'  used to show the 301 and the redirect, but that's fine now.  Best to add `follow_redirects` just in case it happens again though.

Reply:
agreed.